xpra/codecs/nvenc/encoder.pyx

```diff
@@ -1410,10 +1413,15 @@ def get_spec(encoding, colorspace):
     global MAX_SIZE
     max_w, max_h = MAX_SIZE.get(encoding, (4096, 4096))
     has_lossless_mode = colorspace in ("XRGB", "ARGB", "BGRX", "BGRA", "r210") and encoding=="h264"
+    codec_class = Encoder
+    setup_cost = 80
+    if USE_SINGLETON_ENCODER:
+        codec_class = UniqueEncoder
+        setup_cost = 0
```

Comment:
Even with these changes, I doubt that the setup cost is as low as a pure software encoder.

Reply:
This was chosen merely to give strong preference to nvenc over x264 software encoder - I think a small value like 5 or 10 might still keep it preferred over software encoding. Of course the actual setup cost is > 0, but the setup cost for an already initialized singleton encoder would be = 0 when initializing the `UniqueEncoder`